src/SFA.DAS.ProviderApprenticeshipsService.Application/Services/ClientContentApiClient.cs

```diff
@@ -0,0 +1,33 @@
+﻿using SFA.DAS.ProviderApprenticeshipsService.Domain.Interfaces;
+using System.Threading.Tasks;
+using System.Net.Http;
+using SFA.DAS.Authentication.Extensions.Legacy;
+
+namespace SFA.DAS.ProviderApprenticeshipsService.Application.Services
+{
+    public class ClientContentApiClient : ApiClientBase, IClientContentApiClient
+    {
+        private readonly string ApiBaseUrl;
+
+        public ClientContentApiClient(HttpClient client, IContentClientApiConfiguration configuration) : base(client)
+        {
+            ApiBaseUrl = configuration.ApiBaseUrl.EndsWith("/")
+                ? configuration.ApiBaseUrl
+                : configuration.ApiBaseUrl + "/";
+        }
+
+        public async Task<string> Get(string type, string applicationId)
+        {
+            string banner = string.Empty;
+            switch (type)
```

Comment:
no need for the switch on type, we should just pass the type to the api and it will return content for that type and applicationId

Reply:
done